ceph_iscsi_config/client.py

```diff
@@ -189,16 +189,30 @@ def define_client(self):
                              "successfully".format(self.iqn))
             self.change_count += 1
 
+    def try_disable_auth(self, tpg):
+        """
+        Disable authentication (enable ACL mode) if this is the last CHAP user.
+
+        LIO doesn't allow us to mix and match ACLs and auth under a tpg. We
+        only allow ACL mode if there are not CHAP users.
+        """
+
+        for client in tpg.node_acls:
+            if client.chap_userid or client.chap_password:
+                return
+
+        tpg.set_attribute('authentication', '0')
 
     def configure_auth(self, auth_type, credentials):
         """
         Attempt to configure authentication for the client
         :return:
         """
 
-
+        chap_enabled=False
```

Comment:
Just FYI, there seems to be a bug in the web based viewer. There was a tab, but I fixed it and did a push -f. If you clone my branch then it shows spaces. It seems to just be in the web based viewer that is still showing a tab.